aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -0,0 +1,208 @@
+    
+    | Setting                       | Value                                                 |
+    |-------------------------------|-------------------------------------------------------|
+    | **Domain Name**               | *&lt;the domain name of your B2C tenant&gt;*          |
+    | **Application ID**            | *&lt;paste the Application ID from the clipboard&gt;* |
+    | **Sign-up or sign-in policy** | `B2C_1_SiUpIn`                                        |
+    
+    Select **OK** to close the **Change Authentication** dialog. Select **OK** to create the web app.
+
+Visual Studio creates the web API with a controller named *ValuesController.cs* that returns hard-coded values for GET requests.  The class is decorated with the [Authorize attribute](xref:security/authorization/simple), so all requests require authentication.
+
+## Run the web API
+
+In Visual Studio, press **F5** to run the API. Visual Studio launches a browser pointed at the APIs root URL. Note the URL in the address bar, and leave the API running in the background.
+
+> [!NOTE] Since there is no controller defined for the root URL, the browser displays a 404 (page not found) error. This is expected behavior.
+
+## Use Postman to get a token and test the API
+
+[Postman](https://getpostman.com/postman) is a tool for testing web APIs. For this tutorial, Postman will simulate a web app that accesses the web API on the user's behalf.
+
+### Register Postman as a web app
+
+Since Postman is going to simulate a web app that can obtain tokens from the Azure AD B2C tenant, it must be registered in the tenant as a web app.  Register Postman using [the steps in the documentation](/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-app) under the **Register a web app** section. Stop at the **Create a web app client secret** section. A client secret isn't required for this tutorial. 
+
+Use the following values:
+
+| Setting                       | Value                            | Notes                           |
+|-------------------------------|----------------------------------|---------------------------------|
+| **Name**                      | Postman                          |                                 |
+| **Include web app / web API** | Yes                              |                                 |
+| **Allow implicit flow**       | Yes                              |                                 |
+| **Reply URL**                 | `https://getpostman.com/postman` |                                 |
+| **App ID URI**                | *&lt;leave blank&gt;*            | Not required for this tutorial. |
+| **Include native client**     | No                               |                                 |
+
+The newly-registered web app needs permission to access the web API on the user's behalf.  
+
+1. Select **Postman** in the list of apps, and then select **API access** from the menu on the left.
```

Comment:
@guardrex is that your preference or the standard for this docset?  Reason I ask is because, while I know Markdown supports auto-numbering, one of the guiding principles of Markdown is prose that is easily readable in plaintext.  I make a conscious choice to manually number to support that, but I can also see how the auto-numbering makes doc maintenance easier.  Thoughts?

Reply:
my markdownlint extension complains about those but I choose to ignore. I find it so ugly to list 1. 1. 1.... I'd be interested in knowing if there's a rule for docs

Reply:
AFIAK I picked this up from watching earlier authors ... maybe @tdykstra ... maybe others. It was just a convenience to adding something later in the steps not to have to renumber the list. AFAIK it is not a *real* rule. It would be nice tho if it were decided, then we could all just do it the same way ... even if only on a per-repo basis.

Reply:
Some things don't need to be made into rules.  Markdown renumbers when it renders, so it works fine if you go 1.2.3.4. or 1.1.1.1. or whatever. If someone wants to manually renumber everything, that's fine, and if someone else doesn't want to bother, well, that's the kind of convenience Markdown is designed for.